File: Service/Admin/RMA.Service.Admin.MasterDataManager.Contracts/Entities/MedicalReportFormWizardDetail.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

namespace RMA.Service.Admin.MasterDataManager.Contracts.Entities
{
    public class MedicalReportFormWizardDetail
    {
        public int MedicalReportFormWizardDetailId { get; set; }
        public int WorkItemId { get; set; }
        public int? MedicalReportFormId { get; set; }
        public MedicalFormReportTypeEnum? MedicalFormReportType { get; set; }
        public int? WizardId { get; set; }
        public int? DocumentId { get; set; }
        public int PersonEventId { get; set; }
        public string CreatedBy { get; set; }
        public System.DateTime CreatedDate { get; set; }
        public string ModifiedBy { get; set; }
        public System.DateTime ModifiedDate { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/claimcare/src/app/claim-manager/shared/entities/medical-report-form-wizard-detail.ts
import { MedicalFormReportTypeEnum } from 'projects/shared-models-lib/src/lib/enums/medical-form-report-type-enum';

export class MedicalReportFormWizardDetail {
    medicalReportFormWizardDetailId: number;
    workItemId: number;
    medicalReportFormId: number | null;
    medicalFormReportType: MedicalFormReportTypeEnum | null;
    wizardId: number | null;
    documentId: number | null;
    personEventId: number;
    createdBy: string;
    createdDate: Date | string;
    modifiedBy: string;
    modifiedDate: Date | string;
}
